src/content/docs/en/guides/styling.mdx

```diff
@@ -41,11 +41,11 @@ This CSS:
 Compiles to this:
 ```astro
 <style>
-  h1:where(.astro-HHNQFKH6) {
+  h1:where(.astro-hhnqfkh6) {
      color: red;
   }
 
-  .text:where(.astro-HHNQFKH6) {
+  .text:where(.astro-hhnqfkh6) {
     color: blue;
   }
 </style>
```

Comment:
Thanks @natemoo-re!  Just double-triple checking that the `where` here in the default example is still correct, now that the default for scoped styling is `attribute`?

Otherwise, good to go!

Reply:
Great catch!